karma-mocha-old.conf.js.tmpl

```diff
@@ -10,7 +10,7 @@ module.exports = function(config) {
 
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-    frameworks: ['mocha'],
+    frameworks: ['mocha', 'chai-sinon'],
```

Comment:
Yeah, I figured if we're adding new code here we might as well write it in a way that is compatible with the new suite.